test/test.js

```diff
@@ -27,15 +28,14 @@ const publicKeyBCompressed = eccrypto.getPublicCompressed(privateKeyB);
 
 describe("Key conversion", function () {
   it("should allow to convert private key to public", function () {
-    expect(Buffer.isBuffer(publicKey)).to.equal(true);
-    expect(publicKey.toString("hex")).to.equal(
-      "041b84c5567b126440995d3ed5aaba0565d71e1834604819ff9c17f5e9d5dd078f70beaf8f588b541507fed6a642c5ab42dfdf8120a7f639de5122d47a69a8e8d1"
+    // expect(Buffer.isBuffer(publicKey)).to.equal(true);
```

Comment:
did this do anything important? if not, just remove the whole line. no reason to leave the comment.